tests/dummy/app/controllers/application.js

```diff
@@ -192,10 +194,10 @@ export default Ember.Controller.extend({
      */
     _createGroups() {
         const dimensions = this.get('dimensions');
-        const groupNames = ['calls', 'chats', 'emails'];
-        this.set('groups', groupNames.map(name => dimensions.group().reduceSum(item => item[name])));
+        // const groupNames = ['calls', 'chats', 'emails'];
+        // this.set('groups', groupNames.map(name => dimensions.group().reduceSum(item => item[name])));
```

Comment:
^ dead code?

Reply:
It's just a convenient way to switch back and forth between GROUPED and STACKED/LAYERED; comment out the two lines above this and uncomment these lines

Reply:
mind leaving a comment there? Otherwise I'll probably forget we had this conversation and delete it sometime in the future

Reply:
There's one below but I'll put one above the whole block making it more clear